src/libraries/System.Private.CoreLib/src/System/SpanHelpers.Byte.cs

```diff
@@ -370,6 +371,40 @@ public static unsafe int IndexOf(ref byte searchSpace, byte value, int length)
                     }
                 }
             }
+            else if (AdvSimd.Arm64.IsSupported)
+            {
+                if (offset < (nuint)(uint)length)
+                {
+                    lengthToExamine = GetByteVector128SpanLength(offset, length);
+
+                    // Mask to help find the first lane in compareResult that is set.
+                    // MSB 0x10 corresponds to 1st lane, 0x01 corresponds to 0th lane and so forth.
```

Comment:
"1st lane, 0th lane and so forth" made me think if the next one would be minus 1st lane :).  Could you please change wording to "LSB 0x01 corresponds to lane 0, 0x10 - to lane 1, and so on" or similar?

Reply:
Haha. True. Fixed.